### PR TITLE
Update hr-cadences.md

### PR DESCRIPTION
**Pull request by natalie-hub**
Updated header and added NH feedback to cadences

### CODE REVIEW

File: operations/workplace/people/hr-cadences.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ The goal is monthly for managers and quarterly for non-managers. Your HR partner
 
 Manager listening sessions will happen once a month. In addition to the above topics, this is also an opportunity for people managers to unpack questions around people management challenges, concerns, and/or best practices. This feedback will also help drive future Manager training topics.
 
+## New Hire Feedback
+
+People use [New Hire feedback](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost/onboarding#post-onboarding-readiness) to improve the onboarding process as it relates to general company onboarding. People may give suggestions to functional or department lead(s) if relevant feedback exists. New hire feedback cadences:
```
Comment (justinegeffen) [inline operations/workplace/people/hr-cadences.md:23]: Is the "New hire feedback cadences" sentence referring to the following sections?

Comment (natalie-hub) [reply operations/workplace/people/hr-cadences.md:23]: @justinegeffen - yes, should I change something?

Comment (justinegeffen) [reply operations/workplace/people/hr-cadences.md:23]: Nope, just confirming so the structure is correctly configured. :)

Comment (justinegeffen) [reply operations/workplace/people/hr-cadences.md:23]: The structure looks a little different to what I'd anticipate after "New hire feedback cadence" as it goes straight to "enablement survey". Wondering if the headers need to be changed?

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Approving so we can iterate on the content.

**Pull request merged**